regression-test/suites/query_p0/sql_functions/datetime_functions/test_date_function.groovy

```diff
@@ -222,6 +222,14 @@ suite("test_date_function") {
     qt_sql """ select date_add(test_datetime, INTERVAL 2 MINUTE) result from ${tableName}; """
     qt_sql """ select date_add(test_datetime, INTERVAL 2 SECOND) result from ${tableName}; """
 
+    String explainResult
+    explainResult = sql("select * from ${tableName} where test_datetime >= date_add('2024-01-16',INTERVAL 1 day);")
+    assertFalse(explainResult.contains("date_add"))
+    explainResult = sql("select * from ${tableName} where test_datetime >= months_add('2024-01-16',1);")
+    assertFalse(explainResult.contains("months_add"))
+    explainResult = sql("select * from ${tableName} where test_datetime >= years_add('2024-01-16',1);")
+    assertFalse(explainResult.contains("years_add"))
```

Comment:
we should check the result is right, wo need to check contains '2025-01-16'. btw, u should use
```
explain {
    sql """ ... """
    contains "...."
}
```

Reply:
done